Simplenote/Classes/SPNoteEditorViewController+Extensions.swift

```diff
@@ -35,6 +35,9 @@ extension SPNoteEditorViewController {
         checklistButton = UIBarButtonItem(image: .image(name: .checklist), style: .plain, target: self, action: #selector(insertChecklistAction(_:)))
         checklistButton.accessibilityLabel = NSLocalizedString("Inserts a new Checklist Item", comment: "Insert Checklist Button")
 
+        informationButton = UIBarButtonItem(image: .image(name: .info), style: .plain, target: self, action: #selector(noteInformationWasPressed(_:)))
+        informationButton.accessibilityLabel = NSLocalizedString("Information", comment: "Note Information Button (metrics + references)")
```

Comment:
I'm SO HAPPY to see this!!

(we had, up until last week, a 100% manually laid out NavigationBar titleView)!!

Reply:
Yes! This is definitely better 👍